messaging/EAP7-669_artemis_clusters_with_http_load_balancer.adoc

```diff
@@ -0,0 +1,65 @@
+= Support messaging clusters (Artemis Clusters) behind http load balancers
+:author:            Emmanuel Hugonnet
+:email:             ehugonne.email@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          messaging,jms,load-balancer
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-669
```

Comment:
Please create a WFLY and make that the primary issue here. Please update the PR title and commit message accordingly.

Reply:
+1 this is public repository but EAP7 issues are not accessible publicly.

Reply:
https://issues.jboss.org/browse/WFLY-11829